lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -358,6 +358,28 @@ def test_unenroll_non_user_student(self):
             ).exists()
         )
 
+    def test_manage_add_single_invalid_student(self):
+        """enroll a single non valid student
+        """
+        self.make_coach()
+        self.make_ccx()
+        url = reverse(
+            'ccx_manage_student',
+            kwargs={'course_id': self.course.id.to_deprecated_string()}
+        )
+        data = {
+            'student-action': 'add',
+            'student-id': u','.join(["test_wrong_user_name", ]),  # pylint: disable=no-member
+        }
+        response = self.client.post(url, data=data, follow=True)
+        self.assertEqual(response.status_code, 200)
+
+        self.assertTrue('Could not find a user with name or email "test_wrong_user_name" ' in response.content)
```

Comment:
Since this extends django, you can swap the status code and content with this: https://docs.djangoproject.com/en/1.6/topics/testing/tools/#django.test.SimpleTestCase.assertContains